src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -127,119 +126,235 @@ private function getPath(string $resourceShortName, array $operation, bool $coll
-                    ],
-                ];
+        return $this->getPathOperationByMethod($method, $pathOperation, $mimeTypes, $collection, $resourceMetadata, $resourceClass, $resourceShortName, $operationName, $definitions) ?? $pathOperation;
+    }
 
-                return $pathOperation;
-            }
+    /**
+     * @param \ArrayObject     $definitions
+     * @param bool             $collection
+     * @param ResourceMetadata $resourceMetadata
+     * @param string           $resourceClass
+     * @param string           $operationName
+     *
+     * @return string
+     */
+    private function getDefinitionParameterSchema(\ArrayObject &$definitions, bool $collection, ResourceMetadata $resourceMetadata, string $resourceClass, string $operationName) : string
+    {
+        $parameterSerializerContext = $collection ? $resourceMetadata->getCollectionOperationAttribute($operationName, 'normalization_context', null, true) : $resourceMetadata->getItemOperationAttribute($operationName, 'normalization_context', null, true);
+        $parameterDefinition = $this->getDefinitionSchema($resourceClass, $resourceMetadata, $parameterSerializerContext);
+        $parameterDefinitionKey = $resourceMetadata->getShortName().(null !== $parameterSerializerContext ? '_'.md5(implode(',', $parameterSerializerContext['groups'])) : '');
+        $definitions[$parameterDefinitionKey] = $parameterDefinition;
 
-            $pathOperation['summary'] ?? $pathOperation['summary'] = sprintf('Retrieves a %s resource.', $resourceShortName);
-            $pathOperation['parameters'] ?? $pathOperation['parameters'] = [[
-                'name' => 'id',
-                'in' => 'path',
-                'required' => true,
-                'type' => 'integer',
-            ]];
-            $pathOperation['responses'] ?? $pathOperation['responses'] = [
-                '200' => [
-                    'description' => sprintf('%s resource response', $resourceShortName),
-                    'schema' => ['$ref' => sprintf('#/definitions/%s', $resourceShortName)],
-                ],
-                '404' => ['description' => 'Resource not found'],
-            ];
+        return $parameterDefinitionKey;
+    }
 
-            return $pathOperation;
+    /**
+     * @param string           $method
+     * @param \ArrayObject     $pathOperation
+     * @param array            $mimeTypes
+     * @param bool             $collection
+     * @param ResourceMetadata $resourceMetadata
+     * @param string           $resourceClass
+     * @param string           $resourceShortName
+     * @param string           $operationName
+     * @param \ArrayObject     $definitions
+     *
+     * @return \ArrayObject|null
+     */
+    public function getPathOperationByMethod(string $method, \ArrayObject $pathOperation, array $mimeTypes, bool $collection, ResourceMetadata $resourceMetadata, string $resourceClass, string $resourceShortName, string $operationName, \ArrayObject &$definitions)
```

Comment:
Is this method really necessary? A big 👍 to PHP create for each HTTP verb, but this one can be moved to `getPathOperation` (which is almost empty).